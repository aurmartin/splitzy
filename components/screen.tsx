import { SafeAreaView } from "react-native-safe-area-context";
import { Colors } from "@/lib/constants";

export const Screen = ({ children }: { children: React.ReactNode }) => {
  return (
    <SafeAreaView
      style={{ flex: 1, backgroundColor: Colors.background, padding: 8 }}
    >
      {children}
    </SafeAreaView>
  );
};
