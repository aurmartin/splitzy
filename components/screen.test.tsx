import { render } from "@testing-library/react-native";
import * as React from "react";
import { Text } from "react-native";
import { Screen } from "./screen";

describe("Screen", () => {
  it("renders correctly", () => {
    const tree = render(
      <Screen>
        <Text>Test Content</Text>
      </Screen>,
    );
    expect(tree).toMatchSnapshot();
  });
});
