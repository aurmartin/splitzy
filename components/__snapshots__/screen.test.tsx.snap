// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen renders correctly 1`] = `
<RNCSafeAreaView
  edges={
    {
      "bottom": "additive",
      "left": "additive",
      "right": "additive",
      "top": "additive",
    }
  }
  style={
    [
      {
        "backgroundColor": "hsl(200 15% 90%)",
        "flex": 1,
        "padding": 8,
      },
      undefined,
    ]
  }
>
  <Text>
    Test Content
  </Text>
</RNCSafeAreaView>
`;
